compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -145,6 +121,9 @@ public class BIRPackageSymbolEnter {
     private List<BStructureTypeSymbol> structureTypes; // TODO find a better way
     private BStructureTypeSymbol currentStructure = null;
     private LinkedList<Object> compositeStack = new LinkedList<>();
+    private HashMap<BInvokableSymbol, HashSet<String>> undefinedChildrenMap = new HashMap<>();     // Children functions from the same pkg which are not yet defined
+    private HashMap<BInvokableSymbol, HashSet<String>> failedSymbolsMap = new HashMap<>();       // used for debugging
```

Comment:
Shall we do this comments on the op of the line

Reply:
Modified in commit #a0abba10a989aee9ad4c0ab14369cbeff55c8907